examples/README.md

```diff
@@ -1,5 +1,18 @@
 
-# Example Measurements
+## Example Measurements
+
+### Downloading the Example Measurements
+
+Example measurements are available in the `examples` directory of the `measurementlink-python` Git repository. (That's this directory.) You can download the examples by cloning the Git repository or downloading the appropriate source code zip file.
```

Comment:
```suggestion
This directory contains example measurements. You can download the examples by cloning the Git repository or downloading the appropriate source code zip file.
```

Feel free to disagree with my suggestion. I think the `That's this directory` statement is awkward so I'm trying to get rid of it.

Reply:
`examples/README.md` now says "These are example measurements for MeasurementLink 2023 Q3 or later," which is correct no matter whether you are looking at it on GitHub, a local Git clone, or a downloaded copy of `measurementlink-python-examples-1.1.0-dev0.zip`.